### ISSUE TITLE
Fix inconsistencies in ScienceBase organizational collections

### ISSUE THREAD

**Issue by skybristol** [nggdpp/ndc-experiment#2]
The [NDC collection in ScienceBase](https://www.sciencebase.gov/catalog/item/4f4e4760e4b07f02db47dfb4) was set up with a series of child items mostly representing the organizations/institutions across the activity who are contributing collections of physical item descriptions to the catalog. The organizational collections then contain collection items that describe logical groupings of physical items. Some of those collections then contain other child items describing the physical items themselves.

In working through this, the basic goals are to improve on the metadata for organizational items and collection items and then get rid of all of the other items, replacing functionality with a new set online data in a code-driven infrastructure. I'm finding a number of inconsistencies in how the top-level organization items are laid out and populated, making it difficult to run a logical query and pull them out of the system. I started a notebook exploring this to see what all we need to do to make things consistent. This issue tracks the work and decisions, which will mostly play out in metadata improvements to the ScienceBase Items.

**Comment by skybristol** [comment]
To help resolve some of this, I added a specific tag from a new vocabulary to the organizational items in ScienceBase - [ndc_organization](https://www.sciencebase.gov/catalog/items?filter=tags%3D%7B%22scheme%22%3A%22https%3A%2F%2Fwww.sciencebase.gov%2Fvocab%2Fcategory%2FNGGDPP%2Fnggdpp_collection_types%22%2C%22name%22%3A%22ndc_organization%22%7D&max=20&parentId=4f4e4760e4b07f02db47dfb4). That helps to tease out the organizational items for further processing.

**Comment by nlatysh1** [comment]
Fundamental question is what defines a collection. States have expanded the original concept of Tier 1 - State Organization, Tier 2 - State Collection items. States are creating subfolders to organize collections, e.g., https://www.sciencebase.gov/catalog/folder/56853ebce4b0a04ef4933bbf
This is not super common, but a few examples exist.

**Comment by skybristol** [comment]
> Fundamental question is what defines a collection. States have expanded the original concept of Tier 1 - State Organization, Tier 2 - State Collection items. States are creating subfolders to organize collections, e.g., https://www.sciencebase.gov/catalog/folder/56853ebce4b0a04ef4933bbf
> This is not super common, but a few examples exist.

The problem with this is that it creates an over complicated situation and an extra set of ScienceBase dependencies that will make it difficult to unwind from ScienceBase and move to some other system (should we decide to do that). "Folderizing" in ScienceBase is never a good idea when the intent is to create an organization scheme. Rather, tags or other use of properties in ScienceBase should be used.

**Comment by nlatysh1** [comment]
Yes, absolutely, I agree, and just discussed with Data Management WG. ScienceBase folders and sub-folders seem to "speak" to states because the organization structure mimics PCs, which are easy to understand - metadata or NGGDPP metadata schema continues to be a struggle. Tags are great. In the future: if collection level records serve as the primary discovery records - there needs to be a way to apply different tags to separate items within a collection (I'm not sure what child items will be in the future paradigm).

**Comment by skybristol** [comment]
I think the future paradigm is really just moving away from having folks try to use the ScienceBase UI directly to using your custom app (NDC Dashboard v.2) as the vehicle for managing collections. We can then control what ScienceBase actually looks like behind the scenes through whatever business logic is applied in the app. If people want to see something that looks like folders and subfolders of collections, they can get that view in the app, but it will be accomplished through tagging.

One issue we need to work through, though, is how to maintain backward compatibility in the system while moving to a more consistent future situation. I need to do a little more fiddling with a way to run through the entire system to identify items that are actual collections.
